x-pack/plugins/security_solution/public/management/cypress/e2e/endpoints.cy.ts

```diff
@@ -5,20 +5,83 @@
  * 2.0.
  */
 
-import { login } from '../tasks/login';
-import { runEndpointLoaderScript } from '../tasks/run_endpoint_loader';
+import {
+  ISOLATE_HOST_ROUTE_V2,
+  HOST_METADATA_LIST_ROUTE,
+} from '../../../../common/endpoint/constants';
+import { login, loginWithRole, ROLE } from '../tasks/login';
+import { setupLicense } from '../tasks/license';
+import { platinum, gold } from '../fixtures/licenses';
+import { loadEndpointIfNoneExist } from '../tasks/load_endpoint_data';
 
-describe('Endpoints page', () => {
+const loginWithWriteAccess = (url: string) => {
+  loginWithRole(ROLE.analyst_hunter);
+  cy.visit(url);
+};
+
+describe('Endpoint list', () => {
   before(() => {
-    runEndpointLoaderScript();
+    login();
+    loadEndpointIfNoneExist();
   });
 
-  beforeEach(() => {
-    login();
+  describe('Platinum license', () => {
+    beforeEach(() => {
+      setupLicense(platinum);
+    });
+
+    it('should allow isolating endpoint', () => {
+      loginWithWriteAccess('/app/security/administration/endpoints');
```

Comment:
I see several hard coded page url routes in the tests - can we use the `routing` functions that we have in `management/common/routing` to get these paths instead?

Reply:
Yeah, I think we can import this from constants. 👍

Reply:
0a4dcabf9f9e7dccf28ea37f2d946275ed27c8af